client/src/Filter/Filter.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import './Filter.scss';
+import { Select } from 'antd';
+
+const Option = { Select };
+
+export const Filter = () => {
```

Comment:
so the filter, for now, doesn't do anything?

Reply:
Not yet. It will be implemented with redux selectors on next PRs.